lib/src/components/NxFilterInput/__tests__/NxFilterInput.test.tsx

```diff
@@ -96,6 +99,61 @@ describe('NxFilterInput', function() {
     expect(icon).toHaveClassName('nx-icon--filter-icons');
   });
 
+  it('passes an NxButton with the Close icon and nx-btn--clear classname as the suffixContent', function() {
+    const BtnFixture = function() {
+          return shallowComponent().prop('suffixContent');
+        },
+        btn = shallow(<BtnFixture />);
+
+    expect(btn).toExist();
+    expect(btn).toMatchSelector(NxButton);
+    expect(btn.children()).toMatchSelector(Close);
+    expect(btn).toHaveClassName('nx-btn--clear');
+  });
+
+  it('sets the title as "Clear filter" to the Nxbutton when searchIcon is undefined', function() {
```

Comment:
Also test `false` and `null` in this test.

Reply:
Updated here e4cd31e9323b6c71452478b70739de92d89bd170. Thanks.